examples/rosettacode/24_game.nit

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: 24 game
+# SEE: <http://rosettacode.org/wiki/24_game>
+
+redef class Char 
+	fun is_op: Bool do return "-+/*".has(self)
+	fun is_dig: Bool do return [0..9].has(self.to_i)
+end
+
+# Export `operands` and `numbers` from string `operation`
```

Comment:
I read this comment and the method signature and I still have no idea of what this method is doing with what.

What do you mean by `export`? Does this function add something to `numbers` and `operands` depending on the content of `operation`?